client/commonFramework/end.js

```diff
@@ -112,6 +112,7 @@ $(document).ready(function() {
         common.displayTestResults(tests);
         if (solved) {
           common.showCompletion();
+					common.codeStorage.removeStoredValue(common.challengeName);
```

Comment:
heh, looks like FreeCodeCamp uses spaces for indentation, whereas I use tabs. I'm surprised the linter doesn't catch that. I'll fix this and update my pull request.